src/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -36,15 +36,27 @@
     <Compile Include="System\Security\Cryptography\X509Certificates\X509Certificate2UI.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\SafeX509Handles.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\X509Utils.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Text\ValueStringBuilder.cs">
+      <Link>CoreLib\System\Text\ValueStringBuilder.cs</Link>
+    </Compile>
+    <Compile Include="System\Drawing\FontConverter.cs" />
```

Comment:
I noticed that a lot of these look different from desktop.  Looking through the PR history I see that these were ported from mono rather than desktop.  That's a good start but some callers may care about the desktop behavior on Windows /cc @dreddy-work.